Sources/Basic/OutputByteStream.swift

```diff
@@ -409,3 +488,25 @@ public struct Format {
         return TransformedSeparatedListStreamable(items: items, transform: transform, separator: separator)
     }
 }
+
+/// InMemory implementation of OutputByteStream.
```

Comment:
@abertelrud any opinion on the right name for this subclass (or entire class tree)? Ankit and I went back and forth on this, with no clear resolution...